src/maps/components/GroupMap.vue

```diff
@@ -183,7 +183,7 @@ export default {
       return { opacity: this.showOverlay ? 0.5 : 1 }
     },
     placesWithLocation () {
-      return this.places.filter(hasLocation)
+      return this.places.filter(place => place.status === 'active').filter(hasLocation)
```

Comment:
I kind of ignored the sidenav map... I guess it could also just show the subscribed places, but then it would also be nice to show all places on demand. Leave it for now?